lmnet/lmnet/pre_processor.py

```diff
@@ -112,20 +112,23 @@ def resize_with_gt_boxes(image, gt_boxes, size=(256, 256)):
     height = size[0]
 
     resized_image = resize(image, (height, width))
+    resized_gt_boxes = gt_boxes.copy()
 
     scale = [height / origin_height, width / origin_width]
 
     if gt_boxes is not None and len(gt_boxes) != 0:
-        gt_boxes[:, 0] = gt_boxes[:, 0] * scale[1]
-        gt_boxes[:, 1] = gt_boxes[:, 1] * scale[0]
-        gt_boxes[:, 2] = gt_boxes[:, 2] * scale[1]
-        gt_boxes[:, 3] = gt_boxes[:, 3] * scale[0]
+        resized_gt_boxes[:, 0] = gt_boxes[:, 0] * scale[1]
```

Comment:
I think
```
        resized_gt_boxes[:, 0] *= scale[1]
        resized_gt_boxes[:, 1] *= scale[0]
        resized_gt_boxes[:, 2] *= scale[1]
        resized_gt_boxes[:, 3] *= scale[0]
```
is more faster because `gt_boxes[:, 0] * scale[1]` always create memory copy.

Reply:
@ruimashita Thanks for comments! Everytime I could learn some details.... I didn't know `resized_gt_boxes[:, 0] *= scale[1]` can be done in-place. But I find that if we use `*=`, `dtype` of `resized_gt_boxes` has to be `float`. I think we could tolerate this for generics purpose?
I have updated colab note.

Reply:
@yd8534976 
Thanks. Oh,  `resized_gt_boxes[:, 0] *= scale[1]` raise TypeError !
Your current implementation is better.

Reply:
@ruimashita Thanks!